mmdet3d/models/decode_heads/decode_head.py

```diff
@@ -0,0 +1,118 @@
+from abc import ABCMeta, abstractmethod
+from mmcv.cnn import normal_init
+from mmcv.runner import auto_fp16, force_fp32
+from torch import nn as nn
+
+from mmseg.models.builder import build_loss
```

Comment:
We need `CrossEntropy` loss from MMSeg but not MMDet (it's somehow strange for me that they use different implementations).

Reply:
Do you think there will be many places that we need to use something from MMSeg? We should try our best to avoid more dependencies. I just want to make sure it is a necessity in the future.
If necessary, maybe we can make it an optional requirement when installing mmdet3d, and also please add corresponding instructions in the documentation.

Reply:
Very good problem. In this head PR it seems unnecessary, but in the Segmentor3D class, I inherited a lot from Segmentor2D. Besides, we need to use the `eval_hook` from MMSeg so that we evaluate mIoU rather than mAP.